### PR TITLE
Update myApps info for Stanage go live.

### PR DESCRIPTION
**Pull request by jkwmoore**
Updates the myApps section only.

### CODE REVIEW

**Comment by jkwmoore** [comment]
Closes https://github.com/rcgsheffield/sheffield_hpc/issues/1707

**Review by Carldkennedy — APPROVED** [review]
LGTM. Pushed one commit small correction to working elsewhere in page

File: hpc/connecting.rst [hunk 1]
```diff
@@ -3,33 +3,38 @@
...
 
-.. Hint::
+For easier access to HPC resources,  IT Services runs an instance of Oracle Secure Global desktop called myApps to provide web based access to CLI terminals,
+text editors within interactive sessions for the ShARC, Bessemer and Training HPC clusters.
 
-    Usernames to connect with all HPC services will be the same as those you use to login to MUSE :underline-bold:`not` the prefix on your email address.
 
+.. important:: 
+    
+    Access to the Stanage cluster via the myApps service is currently unavailable.
```
Comment (willfurnass) [inline hpc/connecting.rst:12]:
```suggestion
    The myApps service is only accessible on Bessmer and ShARC (_not_ Stanage).
```
as the current proposed wording implies could be available on Stanage shortly.

File: hpc/connecting.rst [hunk 2]
```diff
@@ -3,33 +3,38 @@
...
 
-In order to access ShARC and Bessemer you must set up a `VPN and MFA <https://www.sheffield.ac.uk/it-services/vpn>`_. 
+In order to access the HPC clusters you must set up a `VPN and MFA <https://www.sheffield.ac.uk/it-services/vpn>`_.
```
Comment (willfurnass) [inline hpc/connecting.rst:14]:
```suggestion
In order to access the HPC clusters you must set up a `VPN connection and MFA <https://www.sheffield.ac.uk/it-services/vpn>`_. 
```

File: hpc/connecting.rst [hunk 3]
```diff
@@ -3,33 +3,38 @@
...
+In order to access the HPC clusters you must set up a `VPN and MFA <https://www.sheffield.ac.uk/it-services/vpn>`_. 
 Also see section **Whether/how you can connect** below. 
 
-The web browser method of access to ShARC and Bessemer is provided by the Secure Global Desktop Client. This method works well 
-for most browsers on all the common computing platforms (Linux, Windows, Mac), however we recommend Internet Explorer or Firefox.
+The web browser method of access to ShARC and Bessemer is recommended. This method works well for most browsers on all the common 
+computing platforms (Linux, Windows, Mac), however we recommend Chrome or Firefox.
 
-To login to ShARC or Bessemer click the following link: `Connect via myAPPs Portal <https://myapps.shef.ac.uk/sgd/index.jsp?langSelected=en>`_
+:underline-bold:`How to login to the myApps service`
```
Comment (willfurnass) [inline hpc/connecting.rst:20]: Better to use (sub)section heading here? https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections

File: hpc/connecting.rst (hunk 1 above)
Comment (willfurnass) [reply hpc/connecting.rst:12]: Nope, looks good

**Pull request merged**